task/06-conditions-n-loops-tasks.js

```diff
@@ -166,10 +191,14 @@ function isInsideCircle(circle, point) {
  *   'entente' => null
  */
 function findFirstSingleChar(str) {
-    throw new Error('Not implemented');
+    for( let i = 0; i < str.length; i++) {
+        if (str.indexOf(str[i]) === i && str.lastIndexOf(str[i]) === i) {
```

Comment:
This line can be `if (str.indexOf(str[i]) ===  str.lastIndexOf(str[i])) {`  with the same result